tools/config.sh

```diff
@@ -97,9 +97,9 @@ INSTALLATIONS=(
   vers:   "cat version|<{ver}>"
   where:  "devel build"
 
-  CNTK ver: "2.2" dashver: "<{ver//./-}>"
+  CNTK ver: "2.3" dashver: "<{ver//./-}>"
   url:    "$INSTALLER_URL/CNTK-<{dashver}>-Linux-64bit-CPU-Only.tar.gz"
-  sha256: "98236d98259d881d489a13ecf2d6bc5d45df773feddc2db1f302614b6cb02d64"
+  sha256: "fc3e4e304fc810e93b9a350a80a6872fdc64cd124fd49571bd1ff9297c212f40"
```

Comment:
Can you ensure that this works out of the box on the published JAR? Right now this lacks openCV sos and DLLs.